### PR TITLE
- first attempt at recipe

### PR DESCRIPTION
**Pull request by charman2**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/mesas/meta.yaml [hunk 1]
```diff
@@ -3,19 +3,20 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
 # Using the name variable with the URL in line 14 is convenient
 # when copying and pasting from another recipe, but not really needed.
```
Comment (scopatz) [inline recipes/mesas/meta.yaml]: Please remove comments

File: recipes/mesas/meta.yaml [hunk 2]
```diff
@@ -3,19 +3,20 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
```
Comment (scopatz) [inline recipes/mesas/meta.yaml]: Please copy the example, rather than removing it

Comment (charman2) [reply recipes/mesas/meta.yaml]: Thankyou! Fixed

File: recipes/mesas/meta.yaml (hunk 1 above)
Comment (charman2) [reply recipes/mesas/meta.yaml]: Thankyou! Fixed

File: recipes/example/meta.yaml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
 
 # Jinja variables help maintain the recipe as you'll update the version only here.
-# Using the name variable with the URL in line 14 is convenient
-# when copying and pasting from another recipe, but not really needed.
-{% set name = "simplejson" %}
-{% set version = "3.8.2" %}
+  # Using the name variable with the URL in line 14 is convenient
+  # when copying and pasting from another recipe, but not really needed.
```
Comment (marcelotrevisani) [inline recipes/example/meta.yaml]: could you please rever the example?

File: recipes/mesas/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+  {% set name = "mesas" %}
+    {% set version = "v0.1111a" %}
```
Comment (marcelotrevisani) [inline recipes/mesas/meta.yaml]:
```suggestion
{% set version = "v0.1111a" %}
```

File: recipes/mesas/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
+  {% set name = "mesas" %}
```
Comment (marcelotrevisani) [inline recipes/mesas/meta.yaml]:
```suggestion
{% set name = "mesas" %}
```

File: recipes/example/meta.yaml (hunk 3 above)
Comment (charman2) [reply recipes/example/meta.yaml]: No problem. My IDE was adding auto formatting on commit and adding spaces. Being a bit of a novice I am having trouble just getting git to revert, but the example file should be identical to the original now.

File: recipes/mesas/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,52 @@
+{% set name = "mesas" %}
+{% set version = "v0.1111a" %}
```
Comment (chrisburr) [inline recipes/mesas/meta.yaml]:
```suggestion
{% set version = "0.1111a" %}
```

File: recipes/mesas/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/charman2/{{ name }}/archive/{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/mesas/meta.yaml]:
```suggestion
  url: https://github.com/charman2/{{ name }}/archive/v{{ version }}.tar.gz
```

**Pull request merged**